### DISCUSSION TITLE
Unsupported Android version

### DISCUSSION THREAD

**Discussion by RajPratapSinghVIP** [LSPosed/LSPosed#1847] [Q&A] [answered]
Optimizer wrapper 
Unsupported ( Android version unsatisfied)

Please fix this problem coming in LSPosed zygisk version

Device name HMD Global Nokia (TA-1004)/{Nokia 8}
Please solve this problem in next update 
![Screenshot_20220414-213832](https://user-images.githubusercontent.com/83077548/163430918-20cee53f-348d-4745-98c4-8b0a35486d65.png)

**Comment by yujincheng08** [comment, marked as answer]
Impossible.

Technically.

**Comment by giteshraheja** [comment]
![Screenshot_20230728-090708](https://github.com/LSPosed/LSPosed/assets/25604645/806cf3fb-92ce-490e-9c31-c5f9642043a9)

Same Android 9 oneplus 7.

**Reply by mywalkb** [reply]
Dex Optimizer Wrapper is available from Android 10+
It will NEVER be available on Android 9. NEVER!

**Reply by giteshraheja** [reply]
Actually what does it do?
